astropy/modeling/tests/test_bounding_box.py

```diff
@@ -227,8 +218,11 @@ def test_create(self):
         assert bounding_box._order == 'C'
 
         # Error
-        with pytest.raises(ValueError):
-            self.BoundingDomain(model, order=mk.MagicMock())
+        order = mk.MagicMock()
+        MESSAGE = re.escape("order must be either 'C' (C/python order) or "
```

Comment:
Does it have to match so specifically? Is partial match that doesn't require `re` or mock good enough?

Same question to all other similar occurrences.